gaurav/src/routes/Root.tsx

```diff
@@ -1,29 +1,79 @@
 import Button from "@/components/ui/personal/Button";
-import { IdCard } from "lucide-react";
-import { Outlet, useNavigate } from "react-router-dom";
+import { HomePage } from "@/pages/recipe/HomePage";
+import { Link, Outlet, useNavigate } from "react-router-dom";
 export const Root = () => {
   const navigate = useNavigate();
-  const handleClick = () => {
-    navigate("timelog");
+
+  const styleClass = "w-40 h-30 text-gray-500";
+  // Reset Logic
+  const handleReset = () => {
+    try {
+      localStorage.clear();
+      window.location.reload();
```

Comment:
use location.reload where `location = useLocation()` instead of `window.location.reload()`